samples/rails.md

```diff
@@ -22,7 +22,6 @@ FROM ruby:2.5
 RUN apt-get update -qq && apt-get install -y nodejs postgresql-client
 WORKDIR /myapp
 COPY Gemfile /myapp/Gemfile
-COPY Gemfile.lock /myapp/Gemfile.lock
```

Comment:
The Gemfile.lock could change from day to day, even with the same Gemfile. If the Gemfile.lock changes on another dev's machine, and the change is pulled to my machine, won't WORKDIR sync the Gemfile.lock over to the container?